tests/integration/components/bs-form/element-test.js

```diff
@@ -251,6 +251,19 @@ module('Integration | Component | bs-form/element', function (hooks) {
       assert.dom('[data-test-form-element] > label').hasClass('col-md-3');
       assert.dom('[data-test-form-element] > div').hasClass('col-md-9');
     });
+
+    test('supports horizontal form layout with multiple custom grid classes', async function (assert) {
+      await render(hbs`
+        <BsForm @formLayout="horizontal" as |form|>
+          <form.element @controlType="text" @label="some label" @options={{this.simpleOptions}} @horizontalLabelGridClass="col-md-3 col-lg-2" data-test-form-element />
+        </BsForm>
+      `);
+
+      assert.dom('[data-test-form-element] > label').hasClass('col-md-3');
+      assert.dom('[data-test-form-element] > div').hasClass('col-md-9');
+      assert.dom('[data-test-form-element] > label').hasClass('col-lg-2');
+      assert.dom('[data-test-form-element] > div').hasClass('col-lg-10');
```

Comment:
You could chain them to improve readability:

```suggestion
      assert.dom('[data-test-form-element] > label').hasClass('col-md-3').hasClass('col-lg-2');
      assert.dom('[data-test-form-element] > div').hasClass('col-md-9').hasClass('col-lg-10');
```

Reply:
👍  Added chaining